apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -302,14 +302,28 @@ lookup_client({username, Username}, FormatFun) ->
     lists:append([lookup_client(Node, {username, Username}, FormatFun) || Node <- ekka_mnesia:running_nodes()]).
 
 lookup_client(Node, {clientid, ClientId}, {M,F}) when Node =:= node() ->
-    M:F(ets:lookup(emqx_channel, ClientId));
+    case ets:lookup(emqx_channel, ClientId) of
+        [] ->
+            [];
+        Ls ->
+            lists:concat(lists:map(
```

Comment:
lists:concat will try to format elements as string.

>Concatenates the text representation of the elements of Things. The elements of Things can be atoms, integers, floats, or strings.

maybe `lists:append/1` is what you are looking for ?